### PR TITLE
site updates & refresh

### PR DESCRIPTION
**Pull request by flynnduism**
Proposing an updated design for the site:

* new way of listing projects in the homepage
* typography refreshed
* header menu hides on scroll
* changed up the blog styling
* nicer tag menu to browse blog categories
* changed code snippet color scheme to tomorrow night (more contrast)
* small tweaks and refinements

This replaces a previous wip PR (#45).

---

Screenshots:

Screen Shot 2021-03-14 at 9 14 42 PM

Screen Shot 2021-03-14 at 9 14 30 PM

Screen Shot 2021-03-14 at 9 13 36 PM

Screen Shot 2021-03-14 at 10 09 59 PM

### CODE REVIEW

**Comment by flynnduism** [comment]
Preview site deploy: https://deploy-preview-48--deislabsio.netlify.app/

**Comment by radu-matei** [comment]
Blockquotes are currently centered - would it make sense not to?

image

**Comment by flynnduism** [comment]
> Blockquotes are currently centered - would it make sense not to?

Good spot - yes these should be left-aligned normally, and `justify` aligned for Radu.

**Comment by flynnduism** [comment]
Updated `blockquote` styles. https://deploy-preview-48--deislabsio.netlify.app/posts/a-simple-wasm-linker-js/

Screen Shot 2021-03-26 at 11 53 43 AM

Screen Shot 2021-03-26 at 11 53 53 AM

**Review by radu-matei — APPROVED** [review]

File: content/about.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
 title: "About"
 ---
 
-Deis Labs is a shoutout to Deis, the organization where many of us started our
-journey into Kubernetes and Open Source - creating and nuturing the Helm project
+Deis Labs is a shoutout to [Deis](https://github.com/deis), the organization where many of us started our
+journey into Kubernetes and Open Source - creating and nuturing the [Helm](https://helm.sh/) project
```
Comment (bridgetkromhout) [inline content/about.md:6]: > nuturing

nurturing

Comment (flynnduism) [reply content/about.md:6]: This is much better than sounding like we had Helm spayed

File: themes/fresh/assets/fresh/partials/_responsive.scss [hunk 2]
```diff
@@ -85,7 +86,7 @@ Responsive Styles
     }
 
     .title-wrapper {
-        width: 78vw !important;
+        // width: 78vw !important;
```
Comment (thomastaylor312) [inline themes/fresh/assets/fresh/partials/_responsive.scss:89]: Nit: Looks like a left over debug?

File: themes/fresh/assets/fresh/partials/_responsive.scss [hunk 3]
```diff
@@ -97,7 +98,7 @@ Responsive Styles
 
     .content-blog,
     .content-single {
-        width: 78vw !important;
+        // width: 78vw !important;
```
Comment (thomastaylor312) [inline themes/fresh/assets/fresh/partials/_responsive.scss:101]: Nit: Looks like a left over debug?

**Review by thomastaylor312 — APPROVED** [review]

**Review by technosophos — APPROVED** [review]
Hooray! This is gorgeous!

File: content/about.md (hunk 1 above)
Comment (flynnduism) [reply content/about.md]: Corrected, thanks @bridgetkromhout

File: themes/fresh/assets/fresh/partials/_responsive.scss (hunk 2 above)
Comment (flynnduism) [reply themes/fresh/assets/fresh/partials/_responsive.scss]: Cleaned up, thanks for catching this.

File: themes/fresh/assets/fresh/partials/_responsive.scss (hunk 3 above)
Comment (flynnduism) [reply themes/fresh/assets/fresh/partials/_responsive.scss]: Cleaned up, thanks for catching this.

**Comment by flynnduism** [comment]
Thanks for the feedback. Gonna merge this!

**Pull request merged**